test/extended/ha/ha.go

```diff
@@ -0,0 +1,258 @@
+package ha
+
+/*
+This controllers test suite is not part of the "default" group, because its
+testing involves terminating and launching controllers components of OpenShift.
+Therefore it can't be run in parallel.
+*/
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os/exec"
+	"regexp"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("ha: Election of OpenShift controllers", func() {
+	defer g.GinkgoRecover()
+	var (
+		podConfig  = exutil.FixturePath("fixtures", "hello-world-pod.json")
+		reExitCode = regexp.MustCompile(`(?i)exitcode:(\d+)`)
```

Comment:
There should be another way to get this information.  @mfojtik ?

Also, I don't recognize the string "exitcode".  Is it from our test code?